addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -70,6 +70,25 @@ function factory(dependencies) {
             });
         }
 
+        /**
+         *
+         * @param {mail.message_reaction_group} messageReactionGroup
+         */
+        openReactionsSummary(messageReactionGroup) {
+            if (messageReactionGroup.message) {
+                messageReactionGroup.message.update({
+                    messageReactionsSummary: insertAndReplace({
+                        message: messageReactionGroup.message,
+                        highlightedReaction: replace(messageReactionGroup),
+                    }),
+                });
+                this.threadViewer.update({
+                    hasReactionsSummary: true,
+                    messageReactionGroup: replace(messageReactionGroup),
+                });
+            }
+        }
```

Comment:
avoid boolean to control a view, but create/remove the view relation accordingly